coss/users/forms.py

```diff
@@ -26,7 +26,8 @@ def clean(self, *args, **kwargs):
             error_msg = 'You need to associate this account with a site.'
 
         if ((self.instance.is_superuser or self.instance.is_staff) and
-            cdata.get('username') != self.instance.username and
+            cdata.get('username') != self.instance.username and not
+            cdata.get('password2') and
```

Comment:
Is this supposed to be ``password2`` or is it a typo?

Reply:
It's the field that wagtail passes to the form.